src/compiler/core.ts

```diff
@@ -895,6 +895,14 @@ namespace ts {
         return result;
     }
 
+    export function arrayFromMap<T, U>(iterator: Iterator<T>, f: (value: T) => U) {

```

Comment:
I'd say this is `iteratorToArray` to `arrayFromIterator` since there is nothing map specific in the signature